examples/encfs/README.md

```diff
@@ -177,3 +178,223 @@ Alternatively, the whitelisted command in test-encfs-container outputs the follo
 This is a file inside the filesystem.
 This is a file inside the filesystem.
 ```
+### Step by Step Example On AKS

```

Comment:
I feel like we should have two subfolders for the examples/encfs (and for skr in the future):

-- examples
  -- ENCFS
    -- ACI
    -- AKS
  -- SKR
     -- ACI
    -- AKS

This way you can separate the readmes (it's a long file now) and the required files for each deployment type

Reply:
If we do this, I feel the structure should be: 
--example
  --encfs
     --aci 
     --aks 
@BryceDFisher what's your opinion?

Reply:
If tabs work, let's do that.  If not, we should do subfolders.